### PR TITLE
feat: doppler plus general layout created

### PR DESCRIPTION
**Pull request by GusBaamonde**
- doppler plus general layout created
- the free plan was added to the cards
- improved accordion icons for frequently asked questions

https://cdn.fromdoppler.com/doppler-ui-library/build924/doppler-plus.html

layout-doppler-plus2

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/templates/_doppler-plus.scss [hunk 1]
```diff
@@ -44,4 +67,35 @@
...
+      color: #333;
+    }
+
+    &:before {
+      width: 5px;
+      height: 7px;
+      content: '';
+      left: 15px;
+      top: 4px;
+      border-left: solid 4px #666;
```
Comment (cbernat) [inline assets/scss/templates/_doppler-plus.scss:93]: Color hardcoded 😄

**Comment by cbernat** [comment]
When I start to shrink the layout it turns int left aligned like this:
image
👀

**Comment by GusBaamonde** [comment]
> When I start to shrink the layout it turns int left aligned like this:
> 👀

Can be reviewed in another pr

File: assets/scss/modules/_accordion.scss [hunk 2]
```diff
@@ -31,16 +31,16 @@
         height: 7px;
         content: '';
         left: 0;
-        top: 29px;
-        border-left: solid 6px $dp-color-grey;
-        border-bottom: solid 6px transparent;
-        border-top: solid 6px transparent;
+        top: 31px;
```
Comment (lucianaDilizio) [inline assets/scss/modules/_accordion.scss:34]: What is the reason why is used this kind of numbers? I mean, why not `30px`?

File: assets/scss/modules/_card.scss [hunk 3]
```diff
@@ -54,6 +59,24 @@
       margin: 10px 0 0;
       font-size: 14px;
     }
+
+    .dp-btn-free {
+      justify-content: center;
+      display: flex;
+      align-items: flex-end;
+      height: 170px;
```
Comment (cbernat) [inline assets/scss/modules/_card.scss:67]: Shouldn't all buttons have the same height? or standarized height?

Comment (GusBaamonde) [reply assets/scss/modules/_card.scss:67]: This is not a button it only seems like it, but you're right maybe we can push it with another element 👍

File: assets/scss/modules/_accordion.scss (hunk 2 above)
Comment (GusBaamonde) [reply assets/scss/modules/_accordion.scss:34]: In this case it is a pseudo class for an "icon" in an absolute position, so we can move it to the pixel. In this case we align it to the content. The normalization of spaces that we use in the library is 6px and its multiples, only for paddings and margins. Does not apply in this case.

**Comment by cbernat** [comment]
👍

**Pull request merged**